ppr-api/src/ppr_api/models/utils.py

```diff
@@ -131,6 +130,110 @@
     'SERIAL_NUMBER': 'SS'
 }
 
+# Account financing statement/registration list queries.
+QUERY_ACCOUNT_FINANCING_STATEMENTS = """
+SELECT r.id, r.registration_number, r.registration_ts, r.registration_type, r.registration_type_cl,
+       rt.registration_desc, r.base_reg_number, fs.state_type AS state,
+       CASE WHEN fs.life = 99 THEN -99
+            ELSE CAST(EXTRACT(day from (fs.expire_date - (now() at time zone 'utc'))) AS INT) END expire_days,
+       (SELECT MAX(r2.registration_ts)
+          FROM registrations r2
+         WHERE r2.financing_id = r.financing_id) AS last_update_ts,
+       (SELECT CASE WHEN p.business_name IS NOT NULL THEN p.business_name
+                    WHEN p.branch_id IS NOT NULL THEN (SELECT name FROM client_codes WHERE id = p.branch_id)
+                    ELSE p.last_name || ', ' || p.first_name END
+          FROM parties p
+         WHERE p.registration_id = r.id
+           AND p.party_type = 'RG') AS registering_party,
+       (SELECT CASE WHEN p.business_name IS NOT NULL THEN p.business_name
+                    WHEN p.branch_id IS NOT NULL THEN (SELECT name FROM client_codes WHERE id = p.branch_id)
+                    ELSE p.last_name || ', ' || p.first_name END
+          FROM parties p
+         WHERE p.financing_id = fs.id
+           AND p.registration_id_end IS NULL
+           AND p.party_type = 'SP'
+      ORDER BY p.id desc LIMIT 1) AS secured_party,
+      r.client_reference_id
+  FROM registrations r, registration_types rt, financing_statements fs
+ WHERE r.registration_type = rt.registration_type
+   AND fs.id = r.financing_id
+   AND r.registration_type_cl IN ('PPSALIEN', 'MISCLIEN', 'CROWNLIEN')
+   AND r.account_id = '?'
+   AND (fs.expire_date IS NULL OR fs.expire_date > ((now() at time zone 'utc') - interval '30 days'))
+   AND NOT EXISTS (SELECT r3.id
+                     FROM registrations r3
+                    WHERE r3.financing_id = fs.id
+                      AND r3.registration_type_cl = 'DISCHARGE'
+                      AND r3.registration_ts < ((now() at time zone 'utc') - interval '30 days'))
+ORDER BY r.registration_ts DESC
+FETCH FIRST 1000 ROWS ONLY
```

Comment:
do you want to make these params/envs, so we could alter at runtime without rebuilding the image?

Reply:
Change made in second commit. Added config variables:

    # DB Query limits on result set sizes
    ACCOUNT_REGISTRATIONS_MAX_RESULTS = os.getenv('ACCOUNT_REGISTRATIONS_MAX_RESULTS', '1000')
    ACCOUNT_DRAFTS_MAX_RESULTS = os.getenv('ACCOUNT_DRAFTS_MAX_RESULTS', '1000')
    ACCOUNT_SEARCH_MAX_RESULTS = os.getenv('ACCOUNT_SEARCH_MAX_RESULTS', '1000')